packages/shared/friends/sagas.ts

```diff
@@ -435,6 +438,67 @@ export function getFriendRequests(request: GetFriendRequestsPayload) {
   getUnityInstance().AddFriendRequests(addFriendRequestsPayload)
 }
 
+export async function markAsSeenPrivateChatMessages(userId: string) {
+  const client: SocialAPI | null = getSocialClient(store.getState())
+  if (!client) return
+
+  const socialData: SocialData | undefined = findPrivateMessagingFriendsByUserId(store.getState(), userId)
+  if (!socialData?.conversationId) return
+
+  const ownId = client.getUserId()
+
+  // mark as seen all the messages in the conversation
+  await client.markMessagesAsSeen(socialData.conversationId)
+
+  // get total and user's chat unread messages
+  const unreadMessages = client.getConversationUnreadMessages(socialData.conversationId).length
```

Comment:
How does this work as it seems the previous step was to set every message as seen, do we still get something here?

Reply:
Nope, we're not getting anything from it. Do you think it'd be more appropriate to hardcode the response to the renderer? I mean sth like: 

`  const updateUnseenMessages: UpdateUserUnseenMessagesPayload = {
    userId: ownId,
    total: 0
 }`

`getUnityInstance().UpdateUserUnseenMessages(updateUnseenMessages)`